lib/remote_system.rb

```diff
@@ -140,4 +140,8 @@ def read_file(file)
       raise
     end
   end
+
+  def inject_file(local_file, remote_path)
+    system("scp #{local_file} root@#{host}:#{remote_path}")
```

Comment:
Is there a reason for not using Cheetah (or LoggedCheetah) here? And this command and the `run_helper` one need to be changed to consider the `--remote-user` option.